.github/CODEOWNERS

```diff
@@ -2,3 +2,6 @@
 /.github/ @paritytech/ci @pepoviola
 /.gitlab-ci.yml @paritytech/ci @pepoviola
 /scripts/ci/ @paritytech/ci @pepoviola
```

Comment:
Can we add @emamihe here? Thx!

Reply:
Ah sure. I was not sure if we should annoy him automatically for non infra issues